baybe/searchspace/discrete.py

```diff
@@ -239,6 +239,15 @@ def _on_task_configurations(self) -> pd.Series:
             return pd.Series(True, index=self.exp_rep.index)
         return self.exp_rep[task_param.name].isin(task_param.active_values)
 
+    @property
+    def n_effective_default_kernel_dimensions(self) -> int:
```

Comment:
properties are much further down in the file

Reply:
property removed